File: programovani/ulohy/files/modelovani-obrazcu-i/obdelnik-vnoreny.js
// Deklarace a inicializace    
const a = 6; // Inicializace strany obdelnika
const b = 4; // Inicializace strany obdelnika

// Vnejsi cyklus pro radky
for(var i = 1; i <= b; i++) {

    // Inicalizujeme promennou pro vypsani radky
    var rad = "";

    // Vnitrni cyklus pro jednotlive hvezdicky
    for(var j = 1; j <= a; j++) {            

        // Pridame do textoveho retezce hvezdicku
        rad += "* ";

    }

    // Vypiseme radku
    console.log(rad);

}

File: programovani/ulohy/files/modelovani-obrazcu-i/obdelnik-vnoreny.cpp
#include <iostream>
using namespace std;


int main() {

    // Deklarace a inicializace    
    const int a = 6; // Inicializace strany obdelnika
    const int b = 4; // Inicializace strany obdelnika

    // Vnejsi cyklus pro radky
    for(int i = 1; i <= b; i++) {

        // Vnitrni cyklus pro jednotlive hvezdicky
        for(int j = 1; j <= a; j++) {            

            // Vypiseme hvezdicky
            cout << "* ";

        }

        // Vypiseme novou radku
        cout << endl;

    }

}